awx/main/models/organization.py

```diff
@@ -95,6 +95,9 @@ class Meta:
     job_template_admin_role = ImplicitRoleField(
         parent_role='admin_role',
     )
+    execution_environment_admin_role = ImplicitRoleField(
+        parent_role='admin_role',
+    )
```

Comment:
With the addition of this, I'd expect `access.py` to switch a few things from the organization `admin_role` to the `execution_environment_admin_role`.

https://github.com/ansible/awx/blob/c706bcacc4fdd167e29b23f61c1716bbeac61dd0/awx/main/access.py#L1328

and

https://github.com/ansible/awx/blob/c706bcacc4fdd167e29b23f61c1716bbeac61dd0/awx/main/access.py#L1335

and maybe?

https://github.com/ansible/awx/blob/c706bcacc4fdd167e29b23f61c1716bbeac61dd0/awx/main/access.py#L1342

that one might depend on intent